pkg/kubelet/dockertools/kube_docker_client.go

```diff
@@ -184,7 +185,27 @@ func (d *kubeDockerClient) InspectImage(image string) (*dockertypes.ImageInspect
 		}
 		return nil, err
 	}
-	return &resp, nil
+	parts := strings.SplitN(image, ":", 2)
+	if len(parts) == 1 {
```

Comment:
This is ok, but kubelet applies a default image tag (i.e., "latest") if none is specified.